chapter4/e1-minimum-value.js

```diff
@@ -0,0 +1,3 @@
+function min(a, b) {
+  return (a <= b) ? a : b;
+}
```

Comment:
This looks good, but you should add a few tests for it since that's something a lot of interviewers will want to see. You can just use console logs